src/main/webapp/js/instructor.js

```diff
@@ -14,6 +14,11 @@ $(document).ready(function() {
     
     // bind the show picture onhover events
     bindStudentPhotoHoverLink('.profile-pic-icon-hover');
+    
+    // Formatting searchKey for highlight searchKey plugin ( ['string1','string2',...] )
```

Comment:
This comment doesn't seem to belong here anymore, it should be moved to ~ line 341 since the function name ```highlightSearchResult``` is explanatory enough, and if the contributor wants to know more they can always refer to the comment on the function